scripts/security/certs-create-per-user.sh

```diff
@@ -42,16 +42,17 @@ EOF
 #openssl x509 -noout -text -in $i-ca1-signed.crt
 
 # Sign and import the CA cert into the keystore
-keytool -noprompt -keystore kafka.$i.keystore.jks -alias CARoot -import -file ${CA_PATH}/snakeoil-ca-1.crt -storepass confluent -keypass confluent
+keytool -noprompt -keystore kafka.$i.keystore.jks -alias CARootSnakeoil -import -file ${CA_PATH}/snakeoil-ca-1.crt -storepass confluent -keypass confluent
 #keytool -list -v -keystore kafka.$i.keystore.jks -storepass confluent
 
 # Sign and import the host certificate into the keystore
 keytool -noprompt -keystore kafka.$i.keystore.jks -alias $i -import -file $i-ca1-signed.crt -storepass confluent -keypass confluent -ext "SAN=dns:$i,dns:localhost"
 #keytool -list -v -keystore kafka.$i.keystore.jks -storepass confluent
 
 # Create truststore and import the CA cert
-keytool -noprompt -keystore kafka.$i.truststore.jks -alias CARoot -import -file ${CA_PATH}/snakeoil-ca-1.crt -storepass confluent -keypass confluent
-keytool -noprompt -keystore kafka.$i.truststore.jks -alias ca-bundle -import -file ${CA_PATH}/docker-ca-bundle.crt -storepass confluent -keypass confluent
+keytool -noprompt -keystore kafka.$i.truststore.jks -alias CARootSnakeoil -import -file ${CA_PATH}/snakeoil-ca-1.crt -storepass confluent -keypass confluent
```

Comment:
Note that per https://bugs.openjdk.java.net/browse/JDK-4891485 , keystore aliases are lowercased anyway ... which explains why I was previously seeing `caroot` and not `CARoot`.  Not super important, but might want to not use the camel-case here in case it is surprising or harder to read later.  `snakeoil-caroot` assuming hyphens are OK?

Reply:
@javabrett Thanks.  Fixed in latest commit